### PR TITLE
added some file to it

### PR DESCRIPTION
**Pull request by rahman-sohan**

### CODE REVIEW

**Comment by rahman-sohan** [comment]
Check it

**Comment by rahman-sohan** [comment]
**I reviewed ... and i think now it is well ...**

File: main/static/main/CSS/style.css [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+	height: 54px;
+	border: 1px solid darken(white, 30%);
+	border-radius: 20px 0 0 20px;
+}
+{% load static %}
+
+.container .box-search-in-result .submit {
+	cursor: pointer;
+	border: none;
+	background:  url('{{ STATIC_URL }}main/images/Search-icon.png') no-repeat center center, #1E1E20;
```
Comment (ahadsheikh) [inline main/static/main/CSS/style.css]: this line should be `background:  url("../images/Search-icon.png") no-repeat center center, #1E1E20;`

File: main/static/main/CSS/style.css [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+	max-width: 400px;
+	height: 54px;
+	border: 1px solid darken(white, 30%);
+	border-radius: 20px 0 0 20px;
+}
+
+.submit {
+	cursor: pointer;
+	border: none;
+	background:  url('{{ STATIC_URL }}main/images/Search-icon.png') no-repeat center center, #1E1E20;
```
Comment (ahadsheikh) [inline main/static/main/CSS/style.css]: this line should be `background:  url("../images/Search-icon.png") no-repeat center center, #1E1E20;`

**Review by ahadsheikh — CHANGES_REQUESTED** [review]

**Review by rahman-sohan — COMMENTED** [review]
Everything is al right

**Comment by ahadsheikh** [comment]
Done

**Pull request merged**